### ISSUE TITLE
We're not updating MSU version at all

### ISSUE THREAD

**Issue by Felizolinha** [MegaScratchUserscript/Mega-Scratch-Userscript#27]
MSU version is the same since 16 Nov 2014 [[Commit]](https://github.com/MegaScratchUserscript/Mega-Scratch-Userscript/commit/93687f5ce2460cce029fd838630a86b2bb83b667)!
We need a convention on how we're going to update MSU version, it can't be forgotten like that. There were a lot of updates since November 16th, users might think that the project is abandoned this way :/

**Comment by andrewjcole** [comment]
True! We need to update this more!

**Comment by MrSherlockHolmes** [comment]
That's true. We need fresh ideas.

**Comment by InfiniteDropM** [comment]
MSU could use a fresh rebrand, which could increase the enthusiasm of the project. We could advertise on Scratch, and I could share it with my Twitter followers, who are interested in Scratch. If in-need of any help with designs, etc., I would love to be of assitance.

**Comment by MegaApuTurkUltra** [comment]
Yes, it could use a rebrand. The problem is, most everyone has lost interest in the project (even me!)

I've recently creating a userscript that adds tools to the Scratch forum composer (partially as an April fools joke, but it is mostly legit :P); I'm planning to add it to MSU at some point.

**Comment by andrewjcole** [comment]
I'm still interested.

**Comment by MrSherlockHolmes** [comment]
@andrewjcole What about a system that blocks people who constantly spam you?

**Comment by Yoda3D** [comment]
Yes, we need ideas! xD

**Comment by andrewjcole** [comment]
What about a WYSIWYG editor for the forums?

**Comment by MegaApuTurkUltra** [comment]
Was thinking about that

**Comment by PullJosh** [comment]
http://www.wysibb.com/ ??

**Comment by InfiniteDropM** [comment]
Mhm, that would be good, as the forums need some sort of 'markup' editor alongside.

**Comment by DeleteThisAcount** [comment]
pls come back
